autogen/main.tf

```diff
@@ -149,7 +151,16 @@ locals {
   Get available container engine versions
  *****************************************/
 data "google_container_engine_versions" "region" {
-  provider = "{% if private_cluster %}google-beta{%else %}google{% endif %}"
-  zone     = "${data.google_compute_zones.available.names[0]}"
+  provider = "google-beta"
```

Comment:
What is the motivation for dropping the conditional logic?

Reply:
Same reason as above. In 100% of cases now, we are getting the data for a region which requires the google-beta resource.